GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -21,8 +27,10 @@ fi
 STAGINGDIR=$BUILDOUTPUTDIR"Staging"
 VFSFORGITDESTINATION="usr/local/vfsforgit"
 PRJFSDESTINATION="usr/local/vfsforgit/prjfs"
```

Comment:
nit: If the logger is all that's going in the prjfs folder anymore, I think we could just put it alongside the rest of the product. 

We can fix this up later though (whenever I finish my PR to install the new daemon logger I can also fix this).